src/v/cloud_storage/cache_service.cc

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "cloud_storage/logger.h"
+#include "utils/gate_guard.h"
+#include "vlog.h"
+
+#include <seastar/core/coroutine.hh>
+#include <seastar/core/file.hh>
+#include <seastar/core/fstream.hh>
+#include <seastar/core/future.hh>
+#include <seastar/core/iostream.hh>
+#include <seastar/core/seastar.hh>
+
+#include <bits/types/FILE.h>
+#include <cloud_storage/cache_service.h>
+
+#include <exception>
+#include <optional>
+
+namespace cloud_storage {
+
+cache::cache(
+  std::filesystem::path cache_dir,
+  size_t max_cache_size,
+  ss::lowres_clock::duration check_period)
+  : _cache_dir(std::move(cache_dir))
+  , _max_cache_size(max_cache_size)
+  , _check_period(check_period) {}
+
+ss::future<> cache::start() {
+    // todo: start maintenance cycle here, until then don't delete this print
+    // statements so that compiler is not complaining about unused fields
+    std::cout << "max_cache_size: " << _max_cache_size << "\n";
+    std::cout << "check_period: " << _check_period.count() << "\n";
```

Comment:
you `vlog(...)` to log instead of writing directly to stdout.

Reply:
Fixed.